youtube_dl/extractor/youporn.py

```diff
@@ -146,7 +167,7 @@ def _real_extract(self, url):
             webpage, 'uploader', fatal=False)
         upload_date = unified_strdate(self._html_search_regex(
             [r'Date\s+[Aa]dded:\s*<span>([^<]+)',
-             r'(?s)<div[^>]+class=["\']videoInfo(?:Date|Time)["\'][^>]*>(.+?)</div>'],
+             r'(?s)<div[^>]+class=["\']video-uploaded["\'][^>]*>[^<]+?<span>([^<]+)'],
```

Comment:
Do not remove previous pattern.

Reply:
Alright. As they changed their design, the old pattern fails so i removed it . 
I've now restored the old pattern.

https://regexr.com/4s83s (new pattern)
https://regexr.com/4s83m (old pattern)